### PR TITLE
Improve dependencies and documentation

### PR DESCRIPTION
**Pull request by MarcAntoine-Arnaud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,39 +44,20 @@ and for Ivy:
...
-  </dependencies>
-</project>
-```
-
-Then execute:
+If you need to download all dependencies, you just have to run:
 
 ```
-mvn -f photon-download.pom.xml dependency:copy-dependencies
+gradle getDependencies
```
Comment (rpuri-netflix) [inline README.md]: MarcAntoine-Arnaud, would it make sense to replace this line with "./gradlew getDependencies"

File: build.gradle [hunk 2]
```diff
@@ -80,6 +80,13 @@ test {
     }
 }
 
+repositories { mavenCentral() }
```
Comment (rpuri-netflix) [inline build.gradle]: @MarcAntoine-Arnaud repositories( ) has already been defined at an earlier location in build.gradle. Is there a need to re-define it?

File: README.md (hunk 1 above)
Comment (MarcAntoine-Arnaud) [inline README.md]: Agreed

File: build.gradle (hunk 2 above)
Comment (MarcAntoine-Arnaud) [inline build.gradle]: Of course, it needs to be removed.

**Comment by MarcAntoine-Arnaud** [comment]
@rpuri-netflix I think I have made all updates, thank for the review.

**Comment by rpuri-netflix** [comment]
@MarcAntoine-Arnaud thanks a lot - the pull request has been merged

**Pull request merged**